easybuild/easyconfigs/m/modkit/modkit-0.4.1-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,585 @@
+easyblock = 'Cargo'
+
+name = 'modkit'
+version = '0.4.1'
+
+homepage = 'https://github.com/nanoporetech/modkit'
+description = 'A bioinformatics tool for working with modified bases.'
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/nanoporetech/modkit/archive/']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('Rust', '1.78.0'),
```

Comment:
@pavelToman To fix this:
```
Can't locate IPC/Cmd.pm in @INC
```
You'll need to add `Perl-bundle-CPAN` as a build dep